### ISSUE TITLE
Remove title bar

### ISSUE THREAD

**Issue by Bellavene** [laktak/termplug#1]
Would be perfect if there will be no title bar and no window buttons. Also, would be perfect to add any file preview, not only media files.

**Comment by laktak** [comment]
Yes, removing the title bar would look nice. If you'd like to try you are welcome to submit a PR.

For the file preview it should actually show any file type using the system preview as a fallback.

**Comment by Bellavene** [comment]
Actually, I made another approach. I have linked it to Finder which is then set to Gallery view, and now it previews any file. Ideally would be to make a wrapper for Quicklook without title bar and borders. But I am not smart enough for this.

[Findger](https://github.com/Bellavene/findger)

**Comment by laktak** [comment]
Quicklook is broken for heic images - that's why termplug exists.
